var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -14,59 +14,67 @@ class Esmf(MakefilePackage):
+    defines a common model architecture to support interoperable ESMF components.
+    The NUOPC Layer is included with the ESMF package."""
 
-    homepage = "https://www.earthsystemcog.org/projects/esmf/"
+    homepage = "https://earthsystemmodeling.org/"
     url = "https://github.com/esmf-org/esmf/archive/ESMF_8_0_1.tar.gz"
     git = "https://github.com/esmf-org/esmf.git"
 
-    maintainers("climbfuji", "jedwards4b", "AlexanderRichert-NOAA")
+    maintainers("climbfuji", "jedwards4b", "AlexanderRichert-NOAA", "theurich", "uturuncoglu")
 
     # Develop is a special name for spack and is always considered the newest version
     version("develop", branch="develop")
     # generate chksum with spack checksum esmf@x.y.z
+    version("8.4.1", sha256="1b54cee91aacaa9df400bd284614cbb0257e175f6f3ec9977a2d991ed8aa1af6")
     version("8.4.0", sha256="28531810bf1ae78646cda6494a53d455d194400f19dccd13d6361871de42ed0f")
     version("8.3.1", sha256="6c39261e55dcdf9781cdfa344417b9606f7f961889d5ec626150f992f04f146d")
-    version("8.3.0", sha256="0ff43ede83d1ac6beabd3d5e2a646f7574174b28a48d1b9f2c318a054ba268fd")
-    version("8.3.0b09", commit="5b7e546c4b")
+    version(
+        "8.3.0",
+        sha256="0ff43ede83d1ac6beabd3d5e2a646f7574174b28a48d1b9f2c318a054ba268fd",
+        deprecated=True,
+    )
+    version("8.3.0b09", commit="5b7e546c4b", deprecated=True)
     version("8.2.0", sha256="3693987aba2c8ae8af67a0e222bea4099a48afe09b8d3d334106f9d7fc311485")
     version("8.1.1", sha256="58c2e739356f21a1b32673aa17a713d3c4af9d45d572f4ba9168c357d586dc75")
+    version(
+        "8.1.0",
+        sha256="367dc89ab8e273f6fbe3d63a64a9a6735cbaa77d5f144ebae4af198fee5bb203",
+        deprecated=True,
+    )
     version("8.0.1", sha256="9172fb73f3fe95c8188d889ee72fdadb4f978b1d969e1d8e401e8d106def1d84")
-    version("8.0.0", sha256="051dca45f9803d7e415c0ea146df15ce487fb55f0fce18ca61d96d4dba0c8774")
-    version("7.1.0r", sha256="ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889")
+    version(
+        "8.0.0",
+        sha256="051dca45f9803d7e415c0ea146df15ce487fb55f0fce18ca61d96d4dba0c8774",
+        deprecated=True,
+    )
+    version(
+        "7.1.0r",
+        sha256="ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889",
+        deprecated=True,
+    )
 
     variant("mpi", default=True, description="Build with MPI support")
-    variant("external-lapack", default=False, description="Build with external LAPACK support")
+    variant("external-lapack", default=False, description="Build with external LAPACK library")
     variant("netcdf", default=True, description="Build with NetCDF support")
     variant("pnetcdf", default=True, description="Build with pNetCDF support", when="+mpi")
-    variant("xerces", default=True, description="Build with Xerces support")
+    variant("xerces", default=False, description="Build with Xerces support")
     variant(
-        "parallelio",
+        "external-parallelio",
```

Comment:
I do not think it wise to support the internal pio build in newer versions of esmf where the external pio is available in spack and can be built as a dependancy.

Reply:
@jedwards4b - I agree that 99% of Spack builds would/should use the external ParallelIO dependency. That is why it is the default under ESMF's Spack package.py. However, the option to use internal should still be there for those that want to experiment with it (e.g. @uturuncoglu ).

Reply:
There is a more current thread discussing exactly the same question.. resolve this one to have all the conversation on the other.